### ISSUE TITLE
Code coverage machining leaked into latest beta distribution (v3.0.0-beta.5)

### ISSUE THREAD

**Issue by robations** [node-fetch/node-fetch#802]
E.g. in `node_modules/node-fetch/dist/request.js`:

```
function cov_iv055hrut() {
  var path = "/home/xxczaki/dev/node-fetch/src/request.js";
  var hash = "a968741d2a1e9de35c406f7de97215bad7a83d22";
  var global = new Function("return this")();
  var gcv = "__coverage__";
  var coverageData = {
    path: "/home/xxczaki/dev/node-fetch/src/request.js",
...
```

**Comment by robations** [comment]
FYI @xxczaki :)

**Comment by xxczaki** [comment]
Oh, thanks for the information. I don't really understand how this happened, but I will make sure that the next beta will not contain something like this.

**Comment by robations** [comment]
Pretty sure I've never seen this happen before... I noticed because the node-fetch coverage was appearing in my project and breaking my coverage thresholds. After looking at jest coverage -> babel-plugin-istanbul it might be that NODE_ENV=test when you were running the build, I think it may read this variable.

You might want to deprecate the beta-5 version but will leave that to your discretion:

https://docs.npmjs.com/cli/deprecate

dzieki

**Comment by tinovyatkin** [comment]
This is fixed by #805
